src/public/js/redux/pronunciationCourses.js

```diff
@@ -0,0 +1,41 @@
+import {createAction, handleActions} from 'redux-actions';
+import ajax from '../common/ajax';
+import unionBy from 'lodash/unionBy';
+
+// ------------------------------------
+// Constants
+// ------------------------------------
+export const RECEIVED_PRONUNCIATION_COURSES = 'RECEIVED_PRONUNCIATION_COURSES';
+
+// ------------------------------------
+// Actions
+// ------------------------------------
+export const receivedPronunciationCourses = createAction(RECEIVED_PRONUNCIATION_COURSES, (payload) => payload)
+
+export const actions = {
+  receivedPronunciationCourses,
+  fetchPronunciationCoursesAsync,
```

Comment:
![image](https://cloud.githubusercontent.com/assets/7144958/14226742/a44385c6-f91d-11e5-8ce0-f8db7516204e.png)
"fetchPronunciationCoursesAsync" was used before it was defined